templates/repo/header.tmpl

```diff
@@ -46,7 +46,7 @@
 		</div><!-- end grid -->
 	</div><!-- end container -->
 {{end}}
-{{if not .IsDiffCompare}}
```

Comment:
Why did you remove this check?

Reply:
second image above (in merge description) -> will add first red box